mopidy_mpd/network.py

```diff
@@ -12,6 +12,8 @@
 logger = logging.getLogger(__name__)
 
 
+CONTROL_CHARS =  dict.fromkeys(range(32))
```

Comment:
There should only be one space after the `=` sign.

Reply:
Removed, thank you!